src/org/parosproxy/paros/view/View.java

```diff
@@ -168,6 +175,8 @@
     // ZAP: splash screen
     private SplashScreen splashScreen = null;
 
+    private static Map<AddOn.Status, StatusUI> statusMap = new HashMap<AddOn.Status, StatusUI>();

```

Comment:
I mean remove just the type arguments (i.e. using diamond operator). There shouldn't be a warning because of that.